colorbleed/plugins/maya/publish/validate_animation_out_set_related_node_ids.py

```diff
@@ -3,87 +3,127 @@
 import pyblish.api
 import colorbleed.api
 import colorbleed.maya.lib as lib
-import avalon.io as io
 
 
-class ValidateAnimationOutSetRelatedNodeIds(pyblish.api.InstancePlugin):
-    """Validate rig out_SET nodes have related ids to current context
+def get_parent(node):
+    """Get the parent node of the given node
+    Args:
+        node (str): full path of the node
+
+    Returns:
+        str, full path if parent node
+    """
+    return cmds.listRelatives(node, parent=True, fullPath=True)
 
-    An ID is 'related' if its built in the current Item.
 
-    Note that this doesn't ensure it's from the current Task. An ID created
-    from `lookdev` has the same relation to the Item as one coming from others,
-    like `rigging` or `modeling`.
+class ValidateAnimationOutSetRelatedNodeIds(pyblish.api.InstancePlugin):
```

Comment:
We can shorten this class name. `ValidateOutRelatedNodeIds` or something like that should suffice.